web/src/components/core/logpopup.scss

```diff
@@ -0,0 +1,12 @@
+div.d-installer-logpopup {
+  word-break: break-all;
+  white-space: pre-wrap;
+  font-family: var(--pf-global--FontFamily--monospace);
+  font-size: 80%;
+}
+
+div.d-installer-logline {
+  &:hover {
+    background-color: var(--color-gray-dark);
+  }
+}
```

Comment:
As said in another comment, this could be moved to either, utilities or block CSS files. Let's start with a block by now, something like

```diff
+
+
+.filecontent {
+  font-family: var(--ff-code);
+  font-size: 80%;
+  word-break: break-all;
+  white-space: pre-wrap;
+
+  div:hover {
+    background-color: var(--color-gray-dark);
+  }
+}
```

With a better name for the class, but avoiding the d-installer prefix.